rhc-ose-ansible/roles/openshift-management/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+openshift_token:
+openshift_master_url: localhost
+openshift_login_insecure_flag: --insecure-skip-tls-verify=true
+openshift_login_insecure: False
```

Comment:
Where is this used? Should a conditional use this variable as to whether the `openshift_login_insecure_flag` should be used?